onnxruntime/core/providers/cuda/cuda_execution_provider.cc

```diff
@@ -1840,6 +1840,7 @@ CUDAExecutionProvider::GetCapability(const onnxruntime::GraphViewer& graph,
 
     // none of the provided registries has a CUDA kernel for this node
     if (cuda_kernel_def == nullptr) {
+      LOGS_DEFAULT(WARNING) << "CUDA kernel not supported. Fallback to CPU execution provider for Op type: " << node.OpType() << " node name: " << node.Name();
```

Comment:
This line is duplicated with line 1872. Let's come up with a more unique warning message, so that it's easier to trace through.